<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planets3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/silk/world.png</normaloff>:/icons/silk/world.png</iconset>
  </property>
  <widget class="PlanetsWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Simulation"/>
    <addaction name="actionOpen_Simulation"/>
    <addaction name="actionSave_Simulation"/>
    <addaction name="separator"/>
    <addaction name="actionTake_Screenshot"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuGrid">
     <property name="title">
      <string>Grid</string>
     </property>
     <addaction name="actionOff"/>
     <addaction name="actionLines"/>
     <addaction name="actionPoints"/>
    </widget>
    <addaction name="menuGrid"/>
    <addaction name="actionDraw_Paths"/>
    <addaction name="actionMotion_Blur"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="newPlanet_DockWidget">
   <property name="windowTitle">
    <string>New Planet</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="newPlanet_DockWidgetContents">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="newPosX_Label">
       <property name="text">
        <string>Position X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="newPosX_SpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="newPosY_Label">
       <property name="text">
        <string>Position Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="newPosY_SpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="newPosZ_Label">
       <property name="text">
        <string>Position Z</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="newPosZ_SpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="newVelocityX_Label">
       <property name="text">
        <string>Velocity X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="newVelocityX_SpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="newVelocityY_Label">
       <property name="text">
        <string>Velocity Y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="newVelocityY_SpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="newVelocityZ_Label">
       <property name="text">
        <string>Velocity Z</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="newVelocityZ_SpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="newMass_Label">
       <property name="text">
        <string>Mass</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="newMass_SpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="createPlanet_PushButton">
       <property name="text">
        <string>Create</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/silk/world_add.png</normaloff>:/icons/silk/world_add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="speedControl_DockWidget">
   <property name="windowTitle">
    <string>Speed Control</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="speedControl_DockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDial" name="speed_Dial">
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="notchTarget">
        <double>50.000000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="speedDisplay_lcdNumber">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
       <property name="value" stdset="0">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="PauseResume_Button">
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/silk/control_pause_blue.png</normaloff>:/icons/silk/control_pause_blue.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDelete"/>
   <addaction name="actionClear_Velocity"/>
   <addaction name="actionNew_Simulation"/>
   <addaction name="actionCenter_All"/>
   <addaction name="actionInteractive_Planet_Placement"/>
  </widget>
  <widget class="QDockWidget" name="camera_DockWidget">
   <property name="windowTitle">
    <string>Camera Location</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="camera_DockWidgetContents">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="followPlanetLabel">
       <property name="text">
        <string>Follow Planet</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="followPlanetPushButton">
       <property name="text">
        <string>Currently Selected</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="followCenterLabel">
       <property name="text">
        <string>Follow Center</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="followPlainAveragePushButton">
       <property name="text">
        <string>Plain Average</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="clearFollowPushButton">
       <property name="text">
        <string>Follow None</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="followWeightedAveragePushButton">
       <property name="text">
        <string>Weighted Average</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/cross.png</normaloff>:/icons/silk/cross.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionTake_Screenshot">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/image.png</normaloff>:/icons/silk/image.png</iconset>
   </property>
   <property name="text">
    <string>Take Screenshot</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/world_delete.png</normaloff>:/icons/silk/world_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Selected Planet</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionClear_Velocity">
   <property name="text">
    <string>Clear Velocity</string>
   </property>
   <property name="toolTip">
    <string>Clear the currently selected planet's velocity</string>
   </property>
  </action>
  <action name="actionNew_Simulation">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/asterisk_yellow.png</normaloff>:/icons/silk/asterisk_yellow.png</iconset>
   </property>
   <property name="text">
    <string>New Simulation</string>
   </property>
   <property name="toolTip">
    <string>Start a new simulation with no planets</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCenter_All">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/arrow_inout.png</normaloff>:/icons/silk/arrow_inout.png</iconset>
   </property>
   <property name="text">
    <string>Center All</string>
   </property>
   <property name="toolTip">
    <string>Center all planets.</string>
   </property>
  </action>
  <action name="actionInteractive_Planet_Placement">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/world_add.png</normaloff>:/icons/silk/world_add.png</iconset>
   </property>
   <property name="text">
    <string>Interactive Planet Placement</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionOff">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </action>
  <action name="actionDraw_Paths">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Paths</string>
   </property>
  </action>
  <action name="actionOpen_Simulation">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/folder.png</normaloff>:/icons/silk/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Simulation</string>
   </property>
   <property name="toolTip">
    <string>Open Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Simulation">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/disk.png</normaloff>:/icons/silk/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save Simulation</string>
   </property>
   <property name="toolTip">
    <string>Save Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionMotion_Blur">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Motion Blur</string>
   </property>
   <property name="toolTip">
    <string>Motion Blur</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/information.png</normaloff>:/icons/silk/information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlanetsWidget</class>
   <extends>QWidget</extends>
   <header>planetswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
